### PR TITLE
Added Affinidi to adopters

### PR DESCRIPTION
**Pull request by idestis**

### CODE REVIEW

**Comment by djamaile** [comment]
Welcome 👋 !!

We need the DCO sign off before we can merge it 😄

File: ADOPTERS.md [hunk 1]
```diff
@@ -222,3 +222,4 @@ _You can do this by using the [Adopter form](https://info.backstage.spotify.com/
...
 | [ESW](https://esw.com)                                            | [Alisson Fabiano](https://github.com/afabianoo), [Bruno Quintella](https://github.com/quintelab) and [Guilherme Oenning](https://github.com/goenning) | Backstage is our one stop shop to find everything related to all our services, such as ownership, dependencies, production status, tech health and much more.
 | [FanDuel](https://fanduel.com)                                            | [Diego Herrera](https://github.com/diegoh), [Christy Campbell](https://github.com/FD-ChristopherCampbell) | We use backstage as our developer portal to provide visibility of our software, ownership, strategy, and the state of maturity across disciplines.
+| [Affinidi](https://affinid.com) | [Dmytro Shamenko](https://github.com/idestis), [Denis Fastovets](https://www.linkedin.com/in/denis-fastovets/) | The Backstage is used in the company to enhance development acceleration inside of the company and helps us keep the house in order, from cost efficiency and up to accountability.
```
Comment (freben) [inline ADOPTERS.md:225]: Hey, it seems you forgot the last "i" in the company URL here.

**Comment by idestis** [comment]
> And the DCO again! :) Sorry for the trouble.

done

**Review by djamaile — DISMISSED** [review]

**Review by djamaile — CHANGES_REQUESTED** [review]
Seems like your local formatting tool brought some unnecessarily changes. Lets revert that 😄

**Comment by idestis** [comment]
> Seems like your local formatting tool brought some unnecessarily changes. Lets revert that 😄

My bad, sorry for this. Reverted and fixed only URL in my line with DCO.

File: ADOPTERS.md [hunk 2]
```diff
@@ -222,3 +222,5 @@ _You can do this by using the [Adopter form](https://info.backstage.spotify.com/
...
 | [ESW](https://esw.com)                                            | [Alisson Fabiano](https://github.com/afabianoo), [Bruno Quintella](https://github.com/quintelab) and [Guilherme Oenning](https://github.com/goenning) | Backstage is our one stop shop to find everything related to all our services, such as ownership, dependencies, production status, tech health and much more.
 | [FanDuel](https://fanduel.com)                                            | [Diego Herrera](https://github.com/diegoh), [Christy Campbell](https://github.com/FD-ChristopherCampbell) | We use backstage as our developer portal to provide visibility of our software, ownership, strategy, and the state of maturity across disciplines.
+| [Affinidi](https://affinidi.com) | [Dmytro Shamenko](https://github.com/idestis), [Denis Fastovets](https://www.linkedin.com/in/denis-fastovets/) | The Backstage is used 
+in the company to enhance development acceleration inside of the company and helps us keep the house in order, from cost efficiency and up to accountability.
```
Comment (djamaile) [inline ADOPTERS.md:226]: Almost there! You have a enter here which makes these last lines fall in their own table cell. So, if you do some backspaces then we should be fine 😄

**Review by djamaile — CHANGES_REQUESTED** [review]

**Comment by idestis** [comment]
@djamaile finally I did all changes :)

**Review by djamaile — APPROVED** [review]
Welcome, again :)

**Comment by idestis** [comment]
Now we have to wait for @freben to review additionally. I hope he will like it :)

**Review by freben — APPROVED** [review]
Not just like it, I love it!

**Pull request merged**